spec/features/products_form_spec.rb

```diff
@@ -53,13 +61,29 @@
     ProductDashboard::ATTRIBUTE_TYPES[:release_year] = old_release_year
   end
 
-  it "edits product and meta tag data correctly" do
-    product = create(:product)
+  it "edits product and meta tag data correctly", js: true do
+    product = create(:product, banner: <<~HTML)
+      <div>A banner with a <a href="https://example.com">link</a>.</div>
+    HTML
 
     visit edit_admin_product_path(product)
 
+    within(:rich_text_area, "Banner") do
+      within("div", text: "A banner with a link") do
+        expect(page).to have_link("link", href: "https://example.com")
+      end
+    end
```

Comment:
Ah, perhaps the previous double-within initially came to be as a copy of this one? (Which does need to be double in this case).